Enterprise-CMCS/macpro-mdct-core
# making small readme change in hopes of triggering a GitHub snyk workflow

### Description
This pr is being created because I want to trigger the newly added snyk jira workflow. Upon initial merge to main the workflow was skipped. I believe this was a chicken and egg problem. The workflow needs to exist on the target branch to execute when a PR is created against it. This is to confirm that theory and merge in a small change.


### Related ticket(s)
https://jiraent.cms.gov/browse/CMDCT-2942

---
### How to test
look at the actions run verify the workflow ran, look to see if tickets have been created in jira.


### Important updates


---
### Author checklist


---
